changelog_unreleased/markdown/13320.md

```diff
@@ -0,0 +1 @@
+#### Add `prettier-plugin-nginx` to Community Plugins (#13320 by @joedeandev)
```

Comment:
Normally we don't add changelog for such commit. It doesn't belong to markdown anyway.

Reply:
Noted - commit amended to exclude the offending file.